apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -304,7 +393,7 @@ function AddedSectionsGrid() {
 
             <Box>
                 <Typography variant="h6">{`${scheduleName} (${scheduleUnits} Units)`}</Typography>
-                <Grid container spacing={2} padding={0}>
+                <Box marginBottom={-1}>
```

Comment:
I don't like this negative margin. It feels like something that can cause lots of CSS edge case pains in the future. Could you try to figure out a different way to accomplish this? (i.e. the typography in MUI generally has a lot of vertical spacing, so if you wanted the text to be squished vertically, you could look at modifying the typography directly)